lib/minitest/reporters/mean_time_reporter.rb

```diff
@@ -288,6 +325,58 @@ def rate(run, min, max)
         end
       end
 
+      # @return [Boolean] Whether the given :order option is valid.
+      def valid_order?
+        return true if orders.include?(options[:order])
+
+        fail Minitest::Reporters::MeanTimeReporter::InvalidOrder,
+             '`:order` option must be one of `:desc` or `:asc`'
+      end
+
+      # @return [Boolean] Whether the given :order option is :asc.
+      def order_asc?
+        valid_order? && options[:order] == :asc
+      end
+
+      # @return [Boolean] Whether the given :order option is :desc (default).
+      def order_desc?
+        valid_order? && options[:order] == :desc
+      end
+
+      # @return [Array<Symbol>] The valid options for the :order option.
+      def orders
+        [
+          :desc,
+          :asc,
+        ]
+      end
+
+      # @return [Boolean] Whether the given :sort_column option is valid.
+      def valid_sort_column?
+        return true if sort_columns.include?(options[:sort_column])
+
+        fail Minitest::Reporters::MeanTimeReporter::InvalidSortColumn,
+             '`:sort_column` option must be one of `:avg`, `:min`, `:max` or ' \
```

Comment:
Imho it would be better to use sort_columns to create error message, so we would need to change only one place to add/remove sort columns